pkg/sink/sink_test.go

```diff
@@ -188,13 +240,21 @@ func Test_createResource(t *testing.T) {
 	tests := []struct {
 		name        string
 		resource    json.RawMessage
+		namespace   string
```

Comment:
nit: please rename `namespace` this to `eventListenerNamespace`, because that is the name of the parameter in the `createResource` function.

Also, do you think it would make more sense to name the parameter `defaultNamespace` instead of `eventListenerNamespace`?

Reply:
Either one works for me. I'll use `eventListenerNamespace` for now since `defaultNamespace` seems less immediately clear/information lossy.